### DISCUSSION TITLE
Why just a few of seeders are seeding?

### DISCUSSION THREAD

**Discussion by sepidehab1991** [transmission/transmission#8362] [Support]
### What is the issue?

I have six seeders which have the file and one client which want to download from all of them. All of the available seeders are not seeding just a few of them are seeding. What is the issue?

### Which application of Transmission?

transmission-remote

**Comment by killemov** [comment]
Answering the question with the same question you were asked is not helping. The BitTorrent protocol is quite complex and there could be many reasons seeders are not seeding to you. Maybe your client just stopped asking or maybe the seeders just announce and don't actually seed anything.

**Comment by Pentaphon** [comment]
> I have six seeders which have the file and one client which want to download from all of them. All of the available seeders are not seeding just a few of them are seeding. What is the issue?

This could happen for any reason with peering. Is your port forwarded? If not, you should forward it.

**Comment by sepidehab1991** [comment]
Thank you very much for your reply.

Upon reviewing the log, I encountered the following error: IPv4 DHT not ready (broken, 74 nodes). I tried disabling DHT and manually initiating seeding; however, I was unable to request seeders to connect to the specific client. Additionally, when I set port forwarding to "true," I received a network unreachable (error 101) message—an error that wasn’t present prior to enabling this setting.

Any guidance you could provide would be greatly appreciated.

**Comment by Pentaphon** [comment]
> Additionally, when I set port forwarding to "true," I received a network unreachable (error 101) message—an error that wasn’t present prior to enabling this setting.

Did you open the port in your router?
